homeassistant/components/mqtt/strings.json

```
@@ -392,6 +404,27 @@
                 "transition": "Enable the transition feature for this light"
               }
             },
+            "alarm_control_panel_payload_settings": {
+              "name": "Alarm control payload settings",
```

"panel" is in the key, but not in the actual string?

```suggestion
              "name": "Alarm control panel payload settings",
```

It works in English, but for example in German "Alarm control panel" is just "Alarmzentrale", so we would be diverting a bit more here.